Library/Homebrew/os.rb

```diff
@@ -35,6 +35,14 @@ def self.kernel_version
     @kernel_version ||= Version.new(Utils.safe_popen_read("uname", "-r").chomp)
   end
 
+  # Get the kernel name.
+  #
+  # @api public
+  sig { returns(String) }
+  def self.uname
```

Comment:
```suggestion
  def self.uname_s
```
or
```suggestion
  def self.uname_system
```

Reply:
Given we call `uname -r` "kernel_version". We could call `uname -s` "kernel_name".

Reply:
@Bo98 Naming works for me 👍🏻